ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,65 @@
+
+    ssh my_ocf_username@segfault.ocf.berkeley.edu
+
+If you have trouble connecting please email us at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu).
+
+### The Cluster
+
+Users currently have access to our main HPC server `corruption`, which has the following specifications:
+
+* 2x NVIDIA GTX 1080 Ti
+* 1x Intel Intel(R) Xeon(R) CPU E5-2640 v4
+* 4x 16Gb ECC DDR4-2400 ram
+
+We have also recently been approved a grant that will be awarded in the coming weeks. So in addition to the above, the following will be installed in `corruption`:
+
+* 2x NVIDIA GTX 1080 Ti
+* 1x Intel Intel(R) Xeon(R) CPU E5-2640 v4
+* 12x 16Gb ECC DDR4-2400 ram
+
+### Slurm
+
+We currently use [Slurm][slurm] as our workload manager for the cluster. We will soon post technical documentation about our Slurm configuration, but briefly, Slurm is a free and open-source job scheduler which helps manage multi-user use of distributed computational resources, referred to as nodes. In short, all of your programs will be run through Slurm. To use Slurm there are several commands that will be helpful:
+
+* `srun`: Used to submit jobs.
+* `scontrol`: Used to view and edit the Slurm configuration.
+* `squeue`: Used to view running and queued jobs.
+
+### Dependencies
+For managing application dependencies, one currently has two options:
+
+First you could use a virtual environment if you are using Python packages. To create a virtual environment navigate to your home directory and run the following commands
+
+    virtualenv -p python3.5 venv
+    . venv/bin/activate
+
+This will allow you to *pip install* any Python packages that the OCF does not already have for your program.
+
+For those who need access to non-python dependencies or have already integrated their program into Docker, the second option is to use Singularly containers. [Singularity][singularity] is a containerization platform developed at Laurence Berkeley National Laboratory that is designed specifically for HPC environments. To read more about the benefits of Singularity you can look [Here][singularity_article]. Using a container is quite simple, below are a few commands to help getting started.
+
+1. `singularity build --sandbox ./my_container docker://ubuntu`: This will create a Singularity container named `my_container`. It is important that you use the `--sandbox` option, since this is the only way that you can build a container without root privileges. The `docker://ubuntu` option notifies singularity to bootstrap the container from the official Ubuntu docker container on [Docker Hub][docker_hub]. There is also a [Singularity Hub][singularity_hub], from which you can directly pull Singularity images in a similar fashon. We also have some pre-built containers that you may use to avoid having to build your own. They are currently located at `/home/containers`.
```

Comment:
triple backticks is probably better for the first long singularity command. id say do this for all 3 of the commands